packages/app/cypress/e2e/top-nav.cy.ts

```diff
@@ -431,26 +431,32 @@ describe('Growth Prompts Can Open Automatically', () => {
   })
 
   it('CI prompt auto-opens 4 days after first project opened', () => {
-    cy.intercept('query-HeaderBar_HeaderBarQuery', (req) => {
-      req.on('before:response', (res) => {
-        res.body.data.currentProject.savedState = { firstOpened: 1609459200000,
+    cy.withCtx(
+      (ctx) => {
+        // @ts-ignore sinon is a global in the node process where this is executed
```

Comment:
We don't have to use `ts-ignore` in the launchpad when using sinon, some tsconfig discrepancy I imagine.